core/src/main/java/com/google/zxing/maxicode/decoder/DecodedBitStreamParser.java

```diff
@@ -144,7 +144,7 @@ private static String getPostCode3(byte[] bytes) {
          SETS[0].charAt(getInt(bytes, new byte[] {27, 28, 29, 30, 19, 20})),
          SETS[0].charAt(getInt(bytes, new byte[] {21, 22, 23, 24, 13, 14})),
          SETS[0].charAt(getInt(bytes, new byte[] {15, 16, 17, 18,  7,  8})),
-         SETS[0].charAt(getInt(bytes, new byte[] { 9, 10, 11, 12,  1,  2})),
+         SETS[0].charAt(getInt(bytes, new byte[] { 9, 10, 11, 12,  1,  2}))
```

Comment:
I leave things like this in on purpose. It's really trivial, but if you add a new line, at least you don't have to change this line to add the comma. The symmetry is nice.